recipes/pytest-regressions/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pytest-regressions" %}
+{% set version = "1.0.1" %}
+{% set hash_value = "871b4c784365a9509df1891fbedfd6f3f239d8302468fd987835e093f20ec8c6" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed
+
+requirements:
+  build:
+    - pip
+    - setuptools
+    - setuptools_scm
+
+  run:
+    - pathlib2  # [py27]
+    - pytest-datadir >=1.2.0
+    - pytest >=3.5.0
+    - pyyaml
+    - six
```

Comment:
it seems that it also depends on ``pandas`` and ``numpy`` right?
Because of
https://github.com/ESSS/pytest-regressions/blob/73970d5025649c396442a8bcb1b8bbc3da5f4b69/src/pytest_regressions/num_regression.py#L87

Reply:
you may have to add
```
- {{ pin_compatible('numpy') }}
- pandas
```

Reply:
Actually we want to make the dependency to `pandas` and `numpy` optional, not installed by default. Not sure if there's a way to declare that in `conda build` at the moment.

Will fix the `pin_compatible`, thanks!

Reply:
They are declared as dependencies for tests only, so it seems ok, they are optional already

Reply:
Ah, ok them! :)